### PR TITLE
register attendance

### PR DESCRIPTION
**Pull request by NjaalSoerland**

### CODE REVIEW

File: packages/db/src/types/event.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Generated } from "kysely"
```
Comment (junlarsen) [inline packages/db/src/types/event.ts:1]: This file doesnt exist anymore delete it

File: packages/gateway-trpc/src/modules/event/attendance-router.ts [hunk 2]
```diff
@@ -27,6 +27,18 @@ export const attendanceRouter = t.router({
...
+  registerAttendance: protectedProcedure
+    .input(
+      z.object({
+        userId: z.string(),
+        attendanceId: z.string(),
+        attended: z.boolean(),
+      })
+    )
+    .mutation(async ({ input, ctx }) => {
+      const res = await ctx.attendanceService.registerForAttendance(input.userId, input.attendanceId, input.attended)
```
Comment (junlarsen) [inline packages/gateway-trpc/src/modules/event/attendance-router.ts:39]: inline return

File: packages/db/src/db.generated.d.ts [hunk 3]
```diff
@@ -39,6 +39,7 @@ export interface Attendee {
   updatedAt: Generated<Timestamp>;
   userId: string | null;
   attendanceId: string | null;
+  attended: Generated<boolean>;
```
Comment (junlarsen) [inline packages/db/src/db.generated.d.ts:42]: :heart_eyes:

**Review by junlarsen — CHANGES_REQUESTED** [review]

**Review by BillySBarrett — APPROVED** [review]
very nice :)

**Pull request merged**